doc/reference/configuration/index.rst

```diff
@@ -28,41 +28,112 @@ Command options
 
 .. _index-tarantool_version:
 
-..  option:: -V, --version
+..  option:: -v, --version
 
-    Print product name and version, for example:
+    Print the product name and version.
+
+    **Example**
 
     ..  code-block:: console
 
-        $ ./tarantool --version
-        Tarantool 2.10.4-0-g816000e
-        Target: Darwin-x86_64-Release
+        % tarantool --version
+        Tarantool 2.11.1-0-g96877bd
+        Target: Darwin-arm64-Release
         ...
 
     In this example:
 
-    “Tarantool” is the name of the reusable asynchronous networking
-    programming framework.
+    *   ``2.11.1`` is a Tarantool version.
+        Tarantool follows semantic versioning, which is described in the :ref:`Tarantool release policy <release-policy>` section.
 
-    :ref:`The 3-number version <release-policy>` follows the standard ``<major>-<minor>-<patch>``
-    scheme, in which ``<major>`` number is changed only rarely, ``<minor>`` is
-    incremented for each new milestone and indicates possible incompatible
-    changes, and ``<patch>`` stands for the number of bug fix releases made after
-    the start of the milestone. For non-released versions only, there may be a
-    commit number and commit SHA1 to indicate how much this particular build has
-    diverged from the last release.
+        For non-released versions, there may be a
+        commit number and commit SHA1 to indicate how much this particular build has
+        diverged from the last release.
```

Comment:
Commit number and SHA1 are added to the `--version` output unconditionally, for released versions too.